### PR TITLE
[#5015] Resource view_count fix

### PR DESCRIPTION
**Pull request by pkdash**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [x] Automated Testing

- [x] Sufficient User and Developer Documentation

- [x] Passing Jenkins Build

- [x] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-5018](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-5018)

**Review by devincowan — APPROVED** [review]
thanks Pabitra, one minor suggestion

File: hs_core/models.py [hunk 1]
```diff
@@ -2057,13 +2057,13 @@ class MyResourceContentType(pages.Page, hs_core.AbstractResource):
...
-    def update_view_count(self, request):
+    def update_view_count(self):
         self.view_count += 1
-        self.save(update_fields=["view_count"])
+        type(self).objects.filter(id=self.id).update(view_count=self.view_count)
 
     def update_download_count(self):
         self.download_count += 1
-        self.save(update_fields=["download_count"])
+        type(self).objects.filter(id=self.id).update(download_count=self.download_count)
```
[the remark this answers is not in the document]
Comment (pkdash) [reply hs_core/models.py:2066]: Yeah I should add a comment.

[the remark this answers is not in the document]
Comment (sblack-usu) [reply hs_core/models.py:2066]: nice, I'm impressed you knew that update doesn't trigger the save signal on the model. I didn't believe it, looked it up and sure enough.

**Review by devincowan — APPROVED** [review]

**Pull request merged**